core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1076,6 +1076,32 @@ public static int levenshtein(String string1, String string2) {
     return LEVENSHTEIN_DISTANCE.apply(string1, string2);
   }
 
+  /** SQL FIND_IN_SET(string, stringArray) function. */
+  public static int findInSet(String matchStr, String textStr) {
```

Comment:
Because this method is public, I think we should handle the cases where matchStr and textStr are null

Reply:
For null check issue, we can define the null policy of the function, and null policy will handle null properly in code generation phase, so I think there is no need to implement null check in each function implementation additionally.

Reply:
Agree with @LakeShen.

Check null for public method always valuable. 

And your unit test need have null parameter case.

Reply:
I have added the null check. Thanks for the review.